src/components/TextForm.js

```diff
@@ -20,7 +20,7 @@ export default function TextForm(props) {
   };
 
   const handlePunctuation = () => {
-    let newText = text.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g, "");
+    let newText = text.replace(/[.,\/#!?$%\^&\*;:{}=\-_`~()]/g, "");
```

Comment:
Remove ampersand sign(&) as its a symbol not a punctuation mark.

Reply:
Done within 563e612838c349acc5a1ed4e6729c778f569050b.

Reply:
> Done within [563e612](https://github.com/PCON-Hacktoberfest-2022/Textutils/commit/563e612838c349acc5a1ed4e6729c778f569050b).

Also correct the error of the brackets and duplicate functions which you mentioned.

Reply:
Don't you think it'd be better to do in a separate PR?
I'd suggest pushing this forward and create a new one that won't bring any new stuff but just resolve existing issues.

Reply:
> Don't you think it'd be better to do in a separate PR? I'd suggest pushing this forward and create a new one that won't bring any new stuff but just resolve existing issues.

Okay raise the issue and solve it. Do it fast.